modules/ossm-extensions-ref-wasmplugin.adoc

```diff
@@ -137,7 +137,7 @@ The `PullPolicy` object specifies the pull behavior to be applied when fetching
 |If an existing version of the image has been pulled before, that will be used. If no version of the image is present locally, we will pull the latest version.
 
 |Always
-|We will always pull the latest version of an image when applying this plugin.
+|We will always pull the latest version of an image when applying this plug-in.
```

Comment:
We might not want to use "we"? https://www.ibm.com/docs/en/ibm-style?topic=word-usage#we :D